src/components/Filters/Indicators.tsx

```diff
@@ -10,8 +10,16 @@ interface Child {
     button: Array<any>
 }
 
+interface OutcomeObj {
```

Comment:
We should use the utils type for Stat here instead. https://github.com/HopkinsIDD/covid-dashboard-app/blob/3b13a6b39db9d5fb058570c86046c401d7fc1187/src/utils/constantsTypes.tsx#L5-L9

Not sure if `disabled` is a required field for stat, but you can make it an optional field by adding ? to the end of the field.

```
export interface Stat {
    id: number,
    key: string,
    name: string,
    disabled?: boolean
}
```